lib/cdo/school_autocomplete.rb

```diff
@@ -1,27 +1,33 @@
 require 'singleton'
-require 'triez'
 
 class SchoolAutocomplete
   include Singleton
 
-  def initialize
-    @schools = Triez.new value_type: :object
-    School.all.each do |school|
-      key = "#{school[:name]} #{school[:city]} #{school[:zip]}".downcase
-      @schools.change_all(:suffix, key) {school}
-    end
-  end
-
   def get_matches(query, limit)
```

Comment:
Please add a function description (follow [yard doc format](http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md)), with `@param` entries explaining the query format.